src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -55,7 +55,7 @@ public override ExportResult Export(in Batch<Activity> batch)
                 if (activity.TagObjects.Any())
                 {
                     this.WriteLine("Activity.Tags:");
-                    foreach (var tag in activity.TagObjects)
+                    foreach (var tag in activity.EnumerateTagObjects())
```

Comment:
Would you mind adding `ref readonly` on all of these? Not sure if it really makes a difference, but probably good idea to show best-practice 😄

```csharp
foreach (ref readonly var tag in activity.EnumerateTagObjects())
```

Should apply to all of the "Enumerate" methods.